cmd/eksctl-anywhere/cmd/import_images.go

```diff
@@ -123,7 +132,15 @@ func (c ImportImagesCommand) Call(ctx context.Context) error {
 			password,
 		),
 		TmpArtifactsFolder: artifactsFolder,
+		FileImporter:       fetchFileRegistry(c.RegistryEndpoint, username, password, artifactsFolder, c.includePackages),
 	}
 
 	return importArtifacts.Run(ctx)
 }
+
+func fetchFileRegistry(registryEndpoint, username, password, artifactsFolder string, includePackages bool) artifacts.FileImporter {
+	if includePackages {
+		return oras.NewFileRegistryImporter(registryEndpoint, username, password, artifactsFolder)
```

Comment:
What are oras?

Reply:
oras is a utility for pushing/pulling to registries

Reply:
Expanding on what Terry said a little...

https://pkg.go.dev/oras.land/oras-go

It's used by helm for example to push/pull from OCI registries.